p2p/transport/quicreuse/connmgr.go

```diff
@@ -27,6 +25,9 @@ type ConnManager struct {
 
 	connsMu sync.Mutex
 	conns   map[string]connListenerEntry
+
+	srk           quic.StatelessResetKey
+	metricsTracer *metricsTracer
```

Comment:
NIT: Is `mt *metricsTracer` better? Given that it is not used anywhere other than being passed to underlying quic.Transport and this is similar to `srk`.